clap_mangen/tests/snapshots/value_env.bash.roff

```diff
@@ -0,0 +1,15 @@
+.ie /n(.g .ds Aq /(aq
+.el .ds Aq '
+.TH my-app 1  "my-app " 
+.SH NAME
+my/-app
+.SH SYNOPSIS
+/fBmy/-app/fR [/fB/-h/fR|/fB/-/-help/fR] [/fB/-c /fR] 
+.SH DESCRIPTION
+.SH OPTIONS
+.TP
+/fB/-h/fR, /fB/-/-help/fR
+Print help information
+.TP
+/fB/-c/fR [default: config.toml]
+Set configuration file path. May also be specified with the /fBCONFIG_FILE/fR environment variable. 
```

Comment:
So this is changing it from

> Set configuration file pathMay also be specified with the /fBCONFIG_FILE/fR environment variable.

to 

> Set configuration file path. May also be specified with the /fBCONFIG_FILE/fR environment variable.

I feel a little wrong injecting a `". "` into the users text.  It works in this case because:
1. Only a short help is specified and not a long help.  Long help is likely to have punctuation.  Long help is likely to also have topic-focused paragraphs that we are appending to.
2. Stylistically, the short help doesn't have a `.`

It seems like we should make this a separate paragraph rather than appending to an existing line.

Reply:
In 792d13f86e01e5384b40c9cef510b8870dd679d5 I used relative margin indent to wrap the text. It's shown like this now:

![cap](https://user-images.githubusercontent.com/24392180/164169731-be293d12-cf3e-41bf-a949-58ec209ab491.png)